resources/views/index.php

```diff
@@ -6,23 +6,34 @@
  * @var string $subpage_url
  * @var WP_User $user
  */
-$this->layout( 'layouts/plugin' );
+$this->layout('layouts/plugin');
 ?>
 
 <div>
     <div>
-        <dt-copy-text value="<?php echo esc_url( $magic_link ); ?>"></dt-copy-text>
+        <dt-copy-text value="<?php echo esc_url($magic_link); ?>"></dt-copy-text>
     </div>
 
-    <dt-home-app-grid id="appGrid" app-data='<?php echo esc_attr( htmlspecialchars( $data ) ); ?>'
-                      app-url='<?php echo esc_url( $app_url ); ?>'>
+    <dt-home-app-grid id="appGrid" app-data='<?php echo esc_attr(htmlspecialchars($data)); ?>'
+                      app-url='<?php echo esc_url($app_url); ?>'>
         <!-- Add more app icons as needed -->
     </dt-home-app-grid>
     <div>
-		<?php echo $this->section( 'content' ) ?>
+        <?php echo $this->section('content') ?>
     </div>
 
-    <dt-home-footer translations='<?php echo wp_json_encode( [
-		"hiddenAppsLabel" => __( "Hidden apps", 'dt_home' ),
-	] ) ?>'></dt-home-footer>
+    <dt-home-footer id="hiddenApps"
+                    hidden-data='<?php echo esc_attr(htmlspecialchars($hidden_data)); ?>'
+                    app-url-unhide='<?php echo esc_url($app_url); ?>'></dt-home-footer>
+    <script>
+        document.addEventListener('app-selected', (event) => {
```

Comment:
@ancy2023, this script doesn't seem to do anything. Can we remove it?

Reply:
yes, @incraigulous I have removed and committed.

Reply:
@ancy2023 you can see the issue by clicking "Details" under "All Checks have failed" 

![Screenshot from 2024-02-22 10-00-20](https://github.com/thecodezone/dt-home/assets/5910297/08adcbf5-2ded-4bb5-8fcc-8b2777550ec1)

You can also run PHPCS by running `vendor/bin/phpcs` from your dt-home directory. If this does not work, you may need to `ddev ssh` and then run the command from within ddev.

Reply:
@incraigulous, Thanks It helps me a lot, It workes.

Reply:
@incraigulous, unfortunately, I have committed the app-grid styles files in this branch
app-grid.js and app-icon.js, can you please review those files also merge them?